Sources/LocalReceiptParsing/Helpers/ReceiptStrings.swift

```diff
@@ -28,7 +28,7 @@ enum ReceiptStrings {
     case parsing_receipt
     case refreshing_empty_receipt
     case unable_to_load_receipt(Error)
-    case posting_receipt(AppleReceipt)
+    case posting_receipt(AppleReceipt, initiationSource: String)
```

Comment:
Might be a nit but any reason why this is `String` instead of `InitiationSource`? 🤷‍♂️

Reply:
Good suggestion, changed.

Reply:
Oh I know why: this is used by `ReceiptParser` too, which doesn't have that type